xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/delete.vm

```diff
@@ -228,17 +228,11 @@
     #define($heading)
      <strong>$services.localization.render('core.delete.backlinks')</strong> ($links.size())
     #end
-    #define($message)
-      <ul>
-      #foreach($docname in $links)
-        #set($rdoc = $xwiki.getDocument($docname).getTranslatedDocument())
-        <li><a href="$rdoc.getURL('view')">$escapetool.xml($rdoc.getPlainTitle())</a></li>
-      #end
-      </ul>
-    #end
     #define($body)
       <p class="noitems">
-        $escapetool.xml($services.localization.render('core.delete.backlinksInfo', [$doc.backlinks.size()]))
+        #set ($backLinksURL = $escapetool.xml($doc.getURL('view', 'viewer=backlinks')))
+        $services.localization.render('core.delete.backlinksMessage',
```

Comment:
I think you need HTML escape here (`$escapetool.xml(...)`) done on the results of the call to `render()`.

Reply:
I'm not sure what escaping should be used in order to not broke the link display 
![image](https://user-images.githubusercontent.com/22794181/205949189-d78de6b2-9120-41e2-b9ba-ddcf83efb823.png)

Reply:
You need to first use some non-HTML placeholder and then replace that placeholder after the escaping, see, e.g., here: https://github.com/xwiki/xwiki-platform/blob/5a4155e351dc2fa8adf46a4972587e39ce8cd74b/xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/packagelist.vm#L46-L50

I'm aware this is not nice, we'll work on a better solution.

Reply:
I think this `replaceAll` is not a best practice as this triggers regular expression parsing, this should be `replace`.

Reply:
Thanks @michitux ! Updating it